lms/templates/bookmarks/bookmarks_results.underscore

```diff
@@ -0,0 +1,42 @@
+<div id="my-bookmarks" class="sr-is-focusable" tabindex="-1"></div>
+<h2 class="bookmarks-results-header" tabindex="0"><%= gettext("My Bookmarks") %></h2>
+
+<% if (bookmarks.length) { %>
+
+    <div class='bookmarks-results-list'>
+        <% _.each(bookmarks, function(bookmark, index) { %>
+             <a class="bookmarks-results-list-item" href="<%= bookmark.get('usage_id') %>" aria-describedby="bookmark-type-<%= index %> bookmark-date-<%= index %>">
```

Comment:
What screen reader are you using?  I would like to test with the original solution I suggested in the mockup.  I use 4 different screen readers over here.  They all behave slightly differently, but they all behave badly here.   Without an aria-label or labelledby, the aria-describedby becomes problematic (reads twice).